e2e/periodic/input/simple.nomad

```diff
@@ -0,0 +1,22 @@
+job "periodic" {
+  datacenters = ["dc1"]
+  type        = "batch"
+
+  periodic {
+    cron             = "* * * * *"
+    prohibit_overlap = true
+  }
```

Comment:
The E2E tests run in an environment where there's a Windows machine that can't run Linux Docker images, so we need a constraint to Linux for these jobs.